idd/Energy+.idd.in

```diff
@@ -24861,11 +24866,11 @@ ZoneThermalChimney,
        \memo A thermal chimney is a vertical shaft utilizing solar radiation to enhance natural
        \memo ventilation. It consists of an absorber wall, air gap and glass cover with high solar
        \memo transmissivity.
-   \min-fields 10
+       \min-fields 10
    A1, \field Name
         \required-field
    A2,  \field Zone Name
-        \note Name of zone that is the thermal chimney
+        \note Name of zone that is the thermal chimney.
```

Comment:
Because the thermal chimney is a special zone, only accept a zone name here, not a space name.